### PR TITLE
Add README

### LINKED ISSUE #3: Create a proper README.md
For your next assignment, create a feature branch using the correct naming we discussed and update your README.md file to properly describe how to view your project locally.

Be sure to use Markdown.

Follow the same PR process for this one as well.

### PR DESCRIPTION
**Pull request by NicholasMackeyDev**
closes #3 

This adds a README.md file

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,14 @@
...
+# Montucky
+Montucky is a small repository meant for submission for code review.
+
+## Goal
+The goal is to rebuild the following layout found here: https://www.figma.com/file/T1dmfatKSm53NBentsPIrr/Junior-Mid-FE-Techincal?node-id=0%3A1
+
+## Installation 
+```bash
+git clone https://github.com/NicholasMackeyDev/montucky.git
+```
```
Comment (caseybaggz) [inline README.md]: Once you clone the repo, what do you do after that? 🤔

File: README.md [hunk 2]
```diff
@@ -1,2 +1,14 @@
...
+
+## Contributing
```
Comment (caseybaggz) [inline README.md]: You can remove this - no one is going to contribute to it. Statements like that are meant for libraries, plugins, etc. 😉

File: README.md (hunk 1 above)
Comment (NicholasMackeyDev) [reply README.md]: True. I did not go into much detail. Hopefully this updated one will be better.

File: README.md (hunk 2 above)
Comment (NicholasMackeyDev) [reply README.md]: @caseybaggz 10-4. It was filler, and a poor choice for this README.

Hopefully I did this correctly.

File: README.md [hunk 3]
```diff
@@ -9,6 +9,10 @@ The goal is to rebuild the following layout found here: https://www.figma.com/fi
...
 
-## Contributing 
-Pull requests are welcome. For major changes , please open an 
-issue first to discuss what you would like to change.
+```bash
+cd ./montucky
+```
+
+```bash
+code ./montucky
```
Comment (caseybaggz) [inline README.md]: So close! Let's leave this in for now for the sake of you moving on to the next task.

The takeaway you need to know from this command is:

- This is a VS code specific command - not everyone uses it
- What I was looking for was "navigate to your browser of choice and open `index.html` from your file menu."

Looking forward to your next work! Keep up the good job.

**Pull request merged**